### ISSUE TITLE
Please add WebDAV support

### ISSUE THREAD

**Issue by devcrono** [LightningMods/Itemzflow#74]
I have a NAS server that I can access via WebDAV files.

If you add this function, you will be able to install games or access them directly via the network and internet.

There is an application for PS4 that has this support. See below for documentation if you think it is feasible to implement this

https://github.com/cy33hc/ps4-ezremote-client

Thanks for the great work as always

**Comment by fyngd** [comment]
Yes, I use Synology NAS. I can mount directories using nfs and install pkg, but I cannot mount PS5 folders.

**Comment by LightningMods** [comment]
@fyngd PS5 cannot mount folder rn also we dont currently have any plans to add webdav support, if we do I will reopen this
